packages/angular/package.json

```diff
@@ -61,8 +61,8 @@
     "zone.js": ">=0.11.0"
   },
   "devDependencies": {
-    "@angular-devkit/core": "^14.0.0",
```

Comment:
Unsure if this is related, but I get the following error when I try to run `ionic serve`:

```
liamdebeasi@MacBook-Pro angular-17 % ionic serve
> ng run app:serve --host=localhost --port=8100
[ng] Error: Unknown arguments: host, port

[ERROR] ng has unexpectedly closed (exit code 1).
```

This happens in ng16 and ng17.

Reply:
I don't believe the schematic updates should impact that. This schematic's execution context is purely for the `ng add` command. After that it has no usage. 

For example if I use the dev-build to run `ng add` and then install the latest version of `@ionic/angular` I get the same exception when using `ionic serve`:
```

Developers really should be using the `start` command script with their projects: `npm start`.

Reply:
Ok looked into this more. You cannot assume to use `ionic serve` in a new Angular project. If the project name is not "app" it won't work. 

For example my project is called `angular-blank`, so the command would need to be:

```
ng run angular-blank:serve --host=localhost --port=8100
```

```
> ng run angular-blank:serve --host=localhost --port=8100


Initial Chunk Files | Names         |  Raw Size
polyfills.js        | polyfills     |  82.71 kB | 
styles.css          | styles        |  32.69 kB | 
main.js             | main          |  23.47 kB | 

                    | Initial Total | 138.87 kB

Application bundle generation complete. [2.358 seconds]
Watch mode enabled. Watching for file changes...
  ➜  Local:   http://localhost:8100/
```

Our schematic _could_ change the project name, but I think that is extremely high risk to break a lot of things in the consumers project.

Reply:
If this problem happened before the dev build them I'm fine keeping it out of scope. I assumed you could use `ionic serve` because the `ionic.config.json` file was present, but maybe that's not the case.

Reply:
It looks like you can use the Ionic CLI if you provide the project name for your angular project. If a name is not provided it defaults to `app`.

e.g.:
```
ionic serve --project=angular-blank
```